tests/LibraryLoadThread/native/library.cc

```diff
@@ -0,0 +1,7 @@
+#include <android/log.h>
+
```

Comment:
I think we should also add a `JNI_OnLoad()` export to this file, as part of eventually fixing #7616.  Perhaps it doesn't need to be done before merging, but I also don't want to forget about adding `JNI_OnLoad()` either…

Reply:
I agree, however I need time to dive deeper into how all this stuff with class loaders works, especially in the context of `mono.android.Runtime` so I have it on my TODO, won't forget :)